pkg/serverless/aws/logs.go

```diff
@@ -152,3 +152,15 @@ func (l *LogMessage) UnmarshalJSON(data []byte) error {
 
 	return nil
 }
+
+func ShouldProcessLog(arn string, lastRequestID string, message LogMessage) bool {
+	// If the global request ID or ARN variable isn't set at this point, do not process further
+	if arn == "" || lastRequestID == "" {
+		return false
+	}
+	// Making sure that we do not process empty messages
+	if message.StringRecord == "" && message.Type == "" {
```

Comment:
This change looks good in theory, however some log messages could have no string record but still contain some information. Have we ruled that out?

Reply:
That's why I have added to check against Type as well. 
Those empty messages have no Type. And it seems that log we care about have either type = function or platform
Does it make sense ?